Sources/SwiftUI/ProfileImageView/ProfileImageView.swift

```diff
@@ -142,9 +155,11 @@ private extension ProfileImageView {
     var badgeTapped: (() -> Void)
 
     
-    init?(style: BadgingMode,
-          size: CGSize,
-          badgeTapped: @escaping () -> ()) {
+    init?(
+      style: BadgingMode,
+      size: CGSize,
+      badgeTapped: @escaping () -> ()
+    ) {
```

Comment:
I don't want to negate other developers' code styles.

Reply:
I agree with your point and will revert. But I think we should strive to have a common style throughout the whole package.

Reply:
I agree with @tonikocjan, can we  agree on one approach and enforce it throughout the team?